index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
+      There are many types of digital credential that can be conveyed using
+      this API. Some of these types include:
+    </p>
+    <ul>
+      <li>
+        a driver's license issued by a testing facility
+      </li>
+      <li>
+        a passport issued by a government
+      </li>
+      <li>
+        a student identification card or diploma issued by a university
+      </li>
+      <li>
+        an employee identification card issued by a company
+      </li>
+      <li>
```

Comment:
This falls outside the scope, I think... we want just credentials that related to people.

Reply:
Again, I was just preserving text that already existed:

https://github.com/WICG/digital-identities/pull/82/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L49

If we want to put this out of scope, we can do that, but I know that this is a use case that is of interest for a number of stakeholders that are interested in using this API.

Reply:
how is what in scope being decided/enforced? meaning, will the browser look into the request from the verifier and make a decision whether a requested credential type should be requested from the wallet or not? that sounds like a lot of discretion on the browser api...

Reply:
Right now, I think it will be determined by the supported request protocol/format/query.

Reply:
Reduced the list down to the only credential type we have consensus on right now: 06271a72f5fcfbef716f61d49574b0de9449d5d0. Preserved the full list in https://github.com/WICG/digital-identities/issues/85#issuecomment-1963077832.